Dockerfile

```diff
@@ -8,8 +8,8 @@ ENV LANG=en_US.UTF-8 \
 RUN useradd -d /coreapi coreapi
 
 # https://copr.fedorainfracloud.org/coprs/fche/pcp/
-# https://copr.fedorainfracloud.org/coprs/jpopelka/mercator/
-COPY hack/_copr_fche_pcp.repo hack/_copr_jpopelka-mercator.repo /etc/yum.repos.d/
+# https://copr.fedorainfracloud.org/coprs/msrb/mercator/
+COPY hack/_copr_fche_pcp.repo hack/_copr_msrb-mercator.repo /etc/yum.repos.d/
```

Comment:
Note mercator from my repo will be newer version than what's in Jirka's repo. Therefore this change could break some code, if we use mercator in server somewhere.